roles/sensu-check/defaults/main.yml

```diff
@@ -39,6 +39,15 @@ sensu_checks:
       standalone: true
       command: "check-disk.rb -w 80 -c 90"
       service_owner: openstack
+    check_raid:
+      handler: default
+      interval: 300
+      occurrences: 1
+      standalone: true
+      command: "check-raid.sh -z critical"
```

Comment:
It looks like we went from being able to tweak the criticality level to having to tweak the entire command to change the criticality level. Is that intended?